### PR TITLE
Add missing translatable strings in templates

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: newsletter/forms.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class SubscribeRequestForm(NewsletterForm):
     subscription.
     """
 
-    email_field = forms.EmailField(validators=[validate_email_nouser])
+    email_field = forms.EmailField(label=_("e-mail"), validators=[validate_email_nouser])
```
Comment (dokterbob) [inline newsletter/forms.py]: AFAIK these labels are already translated as part of Django, aren't they?

File: newsletter/templates/newsletter/message/message.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
-{% load thumbnail %}<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN"
-   "http://www.w3.org/TR/html4/strict.dtd">
+{% load thumbnail i18n %}<!DOCTYPE html>
```
Comment (dokterbob) [inline newsletter/templates/newsletter/message/message.html]: +1 on HTML5 headers. Just make sure to state it in the commit message as well.

File: newsletter/templates/newsletter/message/message.txt [hunk 3]
```diff
@@ -12,4 +12,4 @@
 
 ++++++++++++++++++++
 
-Unsubscribe: http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}
+{% trans "Unsubscribe:" %} http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}
```
Comment (dokterbob) [inline newsletter/templates/newsletter/message/message.txt]: Perhaps skip the `:`, that should save an extra string to translate. Most users are expecte to make custom templates anyway.

Comment (claudep) [inline newsletter/templates/newsletter/message/message.txt]: Correct syntax seems more important to me than translator work.

File: newsletter/templates/newsletter/message/message.html (hunk 2 above)
Comment (dokterbob) [inline newsletter/templates/newsletter/message/message.html]: +2 on that suggestion. Maybe finish this work, commit it and then make another commit changing *all* template headers to HTML5.

File: newsletter/forms.py (hunk 1 above)
Comment (dokterbob) [inline newsletter/forms.py]: Perfect then.

File: newsletter/templates/newsletter/message/message.txt (hunk 3 above)
Comment (dokterbob) [inline newsletter/templates/newsletter/message/message.txt]: Why would `{% trans "Unsubscribe:" %}` be better syntax than `{% trans "Unsubscribe" %}:`? Could be I'm missing linguistic subleties here.

Comment (claudep) [inline newsletter/templates/newsletter/message/message.txt]: It's a visual thing, but maybe very personal. Between:
`Unsubscribe http://some.site.com/...`
`Unsubscribe: http://some.site.com/...`
The first one look a bit strange without punctuation before the URL. But if this doesn't disturb you, then fine.

Comment (claudep) [inline newsletter/templates/newsletter/message/message.txt]: Ahhhhh :-) But then you have the problem with languages using a different punctuation to convey the meaning of `:`. You never know with languages...

Comment (dokterbob) [inline newsletter/templates/newsletter/message/message.txt]: But in this case we should also include word order and include the URL in the message as a substitution. I'm not sure if we want to go that detailed, I would like to leave the decision up to you.

**Comment by claudep** [comment]
Thanks for merging. I'm in a busy week, sorry for not following up...

**Comment by dokterbob** [comment]
No prob. I figured no changes need to be made right now. ;)
